### PR TITLE
Support conduit-extra 1.2.0

### PR DESCRIPTION
**Pull request by felixonmars**
Note: Documentation fixes for https://docs.haskellstack.org/en/stable/ should target the "stable" branch, not master.

Please include the following checklist in your PR:

* [x] Any changes that could be relevant to users have been recorded in the ChangeLog.md
* [x] The documentation has been updated, if necessary.

Please also shortly describe how you tested your change. Bonus points for added tests!
Builds and runs fine here.

### CODE REVIEW

File: src/Data/Attoparsec/Interpreter.hs [hunk 1]
```diff
@@ -146,9 +146,9 @@ getInterpreterArgs file = do
 
     decodeError e =
       case e of
-        ParseError ctxs _ (Position line col) ->
+        ParseError ctxs _ (Position line col offset) ->
           if null ctxs
           then "Parse error"
           else ("Expecting " ++ intercalate " or " ctxs)
-          ++ " at line " ++ show line ++ ", column " ++ show col
+          ++ " at line " ++ show line ++ ", column " ++ show col ++ ", offset " ++ show offset
```
Comment (lwm) [inline src/Data/Attoparsec/Interpreter.hs]: Fairly slow as usual but why is the offset useful in this context?

**Comment by lwm** [comment]
Apologies for letting this rot, aside from needed rebase and minor comment, LGTM!

File: src/Data/Attoparsec/Interpreter.hs (hunk 1 above)
Comment (felixonmars) [reply src/Data/Attoparsec/Interpreter.hs]: I am not sure about this either, could remove it if you think it's not relevant :)

Comment (lwm) [reply src/Data/Attoparsec/Interpreter.hs]: Well, do you have an example of the output? Maybe it's useful?

**Comment by mgsloan** [comment]
Yeah, sorry about letting this languish, somehow I missed it.  I don't think offset should be outputted.  Is `Position` a record constructor?  It would be nice if the code would work both with the new version of conduit-combinators and the old, but not strictly necessary.

**Comment by felixonmars** [comment]
Updated to remove offset from output and support both old and new conduit-extra.

File: src/Data/Attoparsec/Interpreter.hs [hunk 2]
```diff
@@ -139,14 +140,18 @@ getInterpreterArgs file = do
 
     parseArgStr str =
       case P.parseOnly (argsParser Escaping) (pack str) of
-        Left err -> handleFailure ("Error parsing command specified in the \
-                        \stack options comment: " ++ err)
+        Left err -> handleFailure ("Error parsing command specified in the "
```
Comment (mgsloan) [inline src/Data/Attoparsec/Interpreter.hs]: Oh man CPP + multiline strings strikes again!  Hah :'(

**Comment by mgsloan** [comment]
LGTM, thanks again!

**Pull request merged**